upup/pkg/fi/cloudup/apply_cluster.go

```diff
@@ -1053,6 +1053,29 @@ func (c *ApplyClusterCmd) validateKubernetesVersion() error {
 		return nil
 	}
 
+	kopsVersion, err := semver.Parse(kopsbase.KOPS_RELEASE_VERSION)
+	if err != nil {
+		klog.Warningf("unable to parse kops version %q", kopsVersion)
+	} else {
+		tooNewVersion := kopsVersion
+		tooNewVersion.Minor += 1
+		tooNewVersion.Pre = nil
+		tooNewVersion.Build = nil
+		if util.IsKubernetesGTE(tooNewVersion.String(), *parsed) {
```

Comment:
I wonder about GT vs GTE.  I think this means that e.g. kops 1.18 won't be able to run k8s 1.19 alpha.1 ?

It's probably fine but you know me - super careful!

Shall we discuss in office hours?

Reply:
I intended for kops 1.18 to balk at running k8s 1.19.0-alpha.1 (unless the environment variable was set)

Reply:
Yeah, I saw that after I commented :-)

It does mean that whatever tests we're still running against k8s HEAD will break at some random point in the cycle when k/k cuts the next alpha.  OTOH that might be a good promopt for us to cut the branch for the next release - though it is typically pretty early in the cycle meaning we'd have to do a _lot_ of cherry picks.

Reply:
FWIW this test-infra PR will fix the prow tests: https://github.com/kubernetes/test-infra/pull/16678/files